### ISSUE TITLE
cifti availability

### ISSUE THREAD

**Issue by oricon** [wayalan/HCPex#1]
Thanks for making this extension available. I work with cifti files and often use the MMP cortical surface parcellation, and the subcortical parcellation will be very useful. Could you make the subcortical parcellation available in a cifti along with the MMP surface parcellation?

**Comment by andreifoldes** [comment]
Yes, this would indeed very useful for running e.g.: **wb_command -cifti-parcellate**
